File: sdk/csharp/ChimeraRuntime.cs
namespace Chimera.Sdk;
public static class ChimeraRuntime {
 public const string Version="1.0.0";
 public static string TargetTriple() => System.Runtime.InteropServices.RuntimeInformation.ProcessArchitecture switch {
 System.Runtime.InteropServices.Architecture.X64=>"chimera-x86_64",
 System.Runtime.InteropServices.Architecture.Arm64=>"chimera-aarch64", _=>"chimera-host"};
}

File: sdk/java/src/org/chimera/sdk/ChimeraRuntime.java
package org.chimera.sdk;
public final class ChimeraRuntime {
 private ChimeraRuntime(){}
 public static String version(){return "1.0.0";}
 public static String targetTriple(){String a=System.getProperty("os.arch","");if(a.equals("amd64")||a.equals("x86_64"))return "chimera-x86_64";if(a.equals("aarch64"))return "chimera-aarch64";return "chimera-host";}
}
